blue_hardware_drivers/src/BLDCControllerClient.cpp

```diff
@@ -289,6 +289,11 @@ void BLDCControllerClient::transmit() {
     sub_packet_buf_.clear();
     // Acquire packet byte format
     auto packet = it->second; // Get packet pointer (value in [key, value] pair)
+    if (packet == NULL) {
+      std::cout << "Generating packet for board: " << (int) it->first << std::endl;
+      throw comms_error("Packet in transmit is null");
```

Comment:
seems like logging the error to `std::cerr` and calling `assert(0)` or something would be more appropriate; this doesn't really feel like a comms error